modules/visual_script/visual_script_editor.cpp

```diff
@@ -2710,6 +2710,11 @@ void VisualScriptEditor::set_debugger_active(bool p_active) {
 	}
 }
 
+CodeEdit *VisualScriptEditor::get_editor() {
+	// Request help, I don't know what to return.
```

Comment:
Compiler require to override this but I never touch `Visual Script` before, so I don't know what to return.

Reply:
Since you are trying to implement a method called `get_current_text_editor`, you shouldn't expose the `CodeEdit *get_editor()` method on `ScriptEditorBase`, and should instead expose it on `ScriptTextEditor`.

In theory, such a method on `ScriptEditorBase` can be useful, but it must return a generic `Control` then, not `CodeEdit`. Either way, `get_current_text_editor` should only return a meaningful result for `ScriptTextEditor`, for others it must return null. An analogous method can be exposed for visual editors that would return its GraphEdit, I guess.